src/Bundle/BundleMetadata.php

```diff
@@ -220,10 +233,12 @@ protected function buildInformation()
             str_replace(':vendor', $this->vendor, $this->configuration['application_dir']).
             DIRECTORY_SEPARATOR.
             $information[1];
-        $this->extendedNamespace = sprintf('%s\\%s',
+        $this->extendedNamespace = sprintf('%s%s\\%s',
```

Comment:
Please take this as an occasion to put the pattern on its own line, it's a PSR2 violation.

Reply:
Should I fix other violations Inside of command as there were more?

Reply:
You can if you want to, in a separate commit though.